ninja-core/src/main/java/ninja/standalone/NinjaConsole.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2018 ninjaframework.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ninja.standalone;
+
+import com.google.inject.Injector;
+import ninja.Bootstrap;
+import ninja.utils.NinjaPropertiesImpl;
+
+public class NinjaConsole extends AbstractConsole<NinjaConsole> {
```

Comment:
There is no test for that class... Up to you if you want to add one...

Reply:
I did have a test, but named it NinjaStandaloneTest rather than what it was testing.  So I renamed it NinjaConsoleTest.